src/main/kotlin/blackjack/domain/BlackJackGame.kt

```diff
@@ -1,80 +1,84 @@
 package blackjack.domain
 
 class BlackJackGame(
-    private val players: List<Player>,
+    private val participants: List<Participant>,
```

Comment:
어쩌면 아래와 같이 설계했을 때 오히려 조건문이 많이 사라지고, 더 구조가 합리적으로 변하는 모습을 보실 수도 있을 것 같습니다. 경철님은 어떻게 생각하시나요?
```suggestion
    private val dealer: Participant,
    private val players: List<Participant>,
```

Reply:
또한 `BlackJackGame`이 현재 하고 있는 일이 다소 많은데요, 로또 미션에서 다뤄졌던 일급 컬렉션 적용도 고려해보시면 어떨까요?
https://jojoldu.tistory.com/412

Reply:
`dealer`를 따로 두는것도 고민을 했는데 위 경우는 변경점이 많아져서 조합 방식으로 딜러와 플레이어를 아우르는 클래스를 두거나 지금 구현과 같이 `Participant` 상위 클래스를 사용한 상속 방식을 사용하는 방법으로 고민했어요.

Reply:
`BlackJackGame`의 주요 역할은 `Participants`를 관리하고 카드를 뽑아서 전달하는 역할인데요. 몇몇 뺄 수 있는 로직은 일급컬렉션으로 빼볼게요.

Reply:
fixed 84b22f95

Reply:
네네 지금 구조도 어떤 의도로 만드셨는지 이해했습니다.

오늘 있을 피드백 수업에서 블랙잭 구조에 대한 여러 가지 설계 아이디어를 다루니, 다른 구현 방법도 참고해보시면 더욱 도움이 되실 것 같아요.